### DISCUSSION TITLE
netdata-updater reset stream.conf ?

### DISCUSSION THREAD

**Discussion by Thibs** [netdata/netdata#18524] [Q&A] [answered]
Hello,

I've recently discovered NetData. I've installed the stable release a few days ago and everything was working fine.

This night, the version was automatically updated to [v1.47.1]

I've discovered my conf file stream.conf was reset due to this upgrade. The [API_KEY]  definition has disappeared

I've restored everything by redefining my API key but I'm really astonish by this.

Is it an usual behavior ?

**Comment by ilyam8** [comment]
Hi, @Thibs.

>  my conf file stream.conf 

Is it in `/etc/netdata`? Can you show the full path to this file?

**Reply by Thibs** [reply, marked as answer]
I think it was just one of my beginner mistake. 

The file I've originally manually edited is /usr/lib/netdata/conf.d/stream.conf

As it was the only existing file, this is the one that was used. It seems normal this one was replaced with package update

As I've now edited it with edit-config, I realize the used file is now /etc/netdata/stream.conf

So there is no bug ... just misunderstanding from me. Sorry for the inconvenience
